boards/atmel/sam0/samd20_xpro/samd20_xpro.yaml

```diff
@@ -9,7 +12,6 @@ toolchain:
 flash: 256
 ram: 32
 supported:
-  - adc
```

Comment:
Were these ADC changes meant to into this commit? If so, please add some details on why in the commit message.

Reply:
Sounds like something that should be submitted as a separate PR?

Reply:
It can be extract on another PR, yes. I'll do it.